app/onepassword_events_api/appserver/static/javascript/components/wizard.js

```diff
@@ -42,14 +59,14 @@ export default function Wizard(props) {
           { className: "description" },
           e("div", { className: "block" }, stepDetails.description)
         ),
-        stepDetails.warning &&
+        currentStep == 1 &&
           (!result.success || result.error) &&
           e(
             "div",
             { className: "warning block" },
             "Your other Splunk apps or add-ons may be able to access your Events API token. Make sure you trust them before you add your token."
           ),
-        stepDetails.redirect &&
+        currentStep == 1 &&
```

Comment:
Previously these conditions were different, is this step supposed to be the `currentStep == 1`?

Reply:
I removed `stepDetails.redirect` (and `stepDetails.warning`) as they're only used for the first step. It was leftover from when the wizard have more steps so I removed them and just hardcode it to the first step.